.ci/Jenkinsfile

```diff
@@ -85,10 +97,23 @@ pipeline {
     }
     post {
         always {
+            script {
+                if(!env.CHANGE_ID) {
```

Comment:
Should probably match up with the above condition exactly

Reply:
In the post stage 'changeRequest()' does not have the desired return value. Tested here and while there was no errors the HTML element for the Snyk Report was not generated. 
Run - https://ci01.dev.r3.com/job/Corda-Gradle-Build-Plugins/job/corda-gradle-plugins/job/release%252F5.1/3/